.cypress/integration/ad/dashboard/ad_dashboard.spec.ts

```diff
@@ -13,31 +13,67 @@
  * permissions and limitations under the License.
  */
 
-/// <reference types="cypress" />
-
 import {
-  AD_PATH,
   AD_URL,
-  API_URL_PREFIX,
   APP_URL_PREFIX,
   DASHBOARD,
-  DETECTORS,
   SLASH,
 } from '../../../utils/constants';
 
 context('AD Dashboard', () => {
-  it('Empty dashboard - no detector', () => {
-    cy.server();
-    cy.route(
-      'GET',
-      [API_URL_PREFIX, AD_PATH, DETECTORS + '*'].join(SLASH),
-      'fixture:no_detector_index_response.json'
-    ).as('getDetectors');
+  it('Empty dashboard - no detector index', () => {
+    cy.mockGetDetectorOnAction('no_detector_index_response.json', () => {
+      cy.visit([APP_URL_PREFIX, AD_URL, DASHBOARD].join(SLASH));
+    });
+    cy.contains('h2', 'You have no detectors');
+  });
 
-    cy.visit([APP_URL_PREFIX, AD_URL, DASHBOARD].join(SLASH));
+  it('Empty dashboard - empty detector index', () => {
+    cy.mockGetDetectorOnAction('empty_detector_index_response.json', () => {
+      cy.visit([APP_URL_PREFIX, AD_URL, DASHBOARD].join(SLASH));
+    });
+    cy.contains('h2', 'You have no detectors');
+  });
 
-    cy.wait('@getDetectors', { requestTimeout: 60_000 });
+  it('AD dashboard - single stopped detector', () => {
+    cy.mockGetDetectorOnAction('single_detector_index_response.json', () => {
+      cy.visit([APP_URL_PREFIX, AD_URL, DASHBOARD].join(SLASH));
+    });
 
-    cy.contains('h2', 'You have no detectors');
+    cy.contains('h3', 'Live anomalies');
+    cy.contains(
+      'p',
+      'All matching detectors are under initialization or stopped for the last 30 minutes. Please adjust filters or come back later.'
+    );
+  });
+
+  it('AD dashboard - create detector', () => {
```

Comment:
should we move this test to a suite pertaining to the create detector page? and maybe just simplify this test to just make sure that when clicking on the create detector button we go to the create detector page?

Reply:
Good point. moved to a separate suite for create detector.